### PR TITLE
some theme fixes

### PR DESCRIPTION
**Pull request by MathiasOhrstrom**

### CODE REVIEW

File: app/src/main/res/layout/activity_marketing.xml [hunk 1]
```diff
@@ -35,6 +35,7 @@
         android:layout_marginEnd="@dimen/base_margin_double"
         android:layout_marginBottom="@dimen/base_margin"
         android:text="@string/MARKETING_SCREEN_CTA"
+        android:textColor="?textColorButton"
```
Comment (hedvigoscar) [inline app/src/main/res/layout/activity_marketing.xml:38]: This should not be necessary, this color should be applied automatically through `style="?materialButtonContainedLargeStyle"`

File: app/src/main/res/layout/dialog_change_start_date.xml [hunk 2]
```diff
@@ -3,6 +3,7 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tool="http://schemas.android.com/tools"
     android:layout_width="match_parent"
+    android:theme="@style/Hedvig.Theme"
```
Comment (hedvigoscar) [inline app/src/main/res/layout/dialog_change_start_date.xml:6]: Is this the optimal way to apply the theme should be applied? Preferably, our theme would automatically be applied to all bottom sheets.

File: app/src/main/res/values/button_themes.xml [hunk 3]
```diff
@@ -12,7 +12,9 @@
         <item name="shapeAppearance">?shapeAppearanceLargeComponent</item>
     </style>
 
-    <style name="Hedvig.Button.Text" parent="Widget.MaterialComponents.Button.TextButton" />
+    <style name="Hedvig.Button.Text" parent="Widget.MaterialComponents.Button.TextButton">
```
Comment (hedvigoscar) [inline app/src/main/res/values/button_themes.xml:15]: Is this correct? (I don't know the answer - double-check with Zak 🙂)

File: app/src/main/res/layout/activity_marketing.xml (hunk 1 above)
Comment (MathiasOhrstrom) [reply app/src/main/res/layout/activity_marketing.xml:38]: yea, i thought so too but i does not seem to work, but i get a white text when on light theme even if we have `<item name="textColorButton">@color/hedvig_black</item>`

Comment (hedvigoscar) [reply app/src/main/res/layout/activity_marketing.xml:38]: This is actually intended, check out the `ThemeOverlay` applied to the root of this Activity. Basically, this screen only has white text no matter what in accordance with a decision from design.

File: app/src/main/res/layout/dialog_change_start_date.xml (hunk 2 above)
Comment (MathiasOhrstrom) [reply app/src/main/res/layout/dialog_change_start_date.xml:6]: dont know why i did that since that is what is does 🤷‍♀

**Pull request merged**